File: src/main/java/com/github/duc010298/web_api/entity/socket/LocationMessage.java
package com.github.duc010298.web_api.entity.socket;

import java.util.Date;

public class LocationMessage {
	private String locationId;
	private double latitude;
	private double longitude;
	private Date timeTracking;
	
	public String getLocationId() {
		return locationId;
	}
	public void setLocationId(String locationId) {
		this.locationId = locationId;
	}
	public double getLatitude() {
		return latitude;
	}
	public void setLatitude(double latitude) {
		this.latitude = latitude;
	}
	public double getLongitude() {
		return longitude;
	}
	public void setLongitude(double longitude) {
		this.longitude = longitude;
	}
	public Date getTimeTracking() {
		return timeTracking;
	}
	public void setTimeTracking(Date timeTracking) {
		this.timeTracking = timeTracking;
	}
}

File: winform_client/winform_client/Entity/LocationMessage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace winform_client.Entity
{
    class LocationMessage
    {
        public string locationId { get; set; }
        public double latitude { get; set; }
        public double longitude { get; set; }
        public DateTime timeTracking { get; set; }
    }
}
